catalog/spatial/registry/registry-policy-plugin/src/test/java/org/codice/ddf/registry/policy/RegistryPolicyPluginTest.java

```diff
@@ -206,6 +211,39 @@ public void testNoRegistryBypassPermissions() throws Exception {
     }
 
     @Test
+    public void testSecurityValueSet() throws Exception {
```

Comment:
testCudRegistryOperations calls into the method and does not generate a security result. However I did rely on the front end input verification as well as an attribute groomer to prevent an empty string from being passed in. I will add a test to make sure this case doesn't result in any NPEs should those elements change.